### PR TITLE
Nicolas/notebook classification

### PR DESCRIPTION
**Pull request by NicolasDurrande**
Nothing super fancy here: just a new notebook on classif + minor edits to the regression one so that they have a similar structure

### CODE REVIEW

File: doc/source/notebooks/manual/basics/classification.ipynb [hunk 1]
```diff
@@ -4,11 +4,299 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m = gpflow.models.VGP(X, Y,\n",
+    "                      likelihood=gpflow.likelihoods.Bernoulli(),\n",
+    "                      kern=gpflow.kernels.Matern52(1))\n",
+    "\n",
+    "o = gpflow.train.ScipyOptimizer()\n",
+    "o.minimize(m);"
```
Comment (vdutor) [inline doc/source/notebooks/manual/basics/classification.ipynb]: can you add a `maxiter` arg to the optimiser please
```
from gpflow.test_util import notebook_niter
o.minimize(m, maxiter=notebook_niter(10e3))
```

File: doc/source/notebooks/manual/basics/classification.ipynb [hunk 2]
```diff
@@ -4,11 +4,299 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
```
Comment (vdutor) [inline doc/source/notebooks/manual/basics/classification.ipynb]: you are not using `tf`, can be removed :)

File: doc/source/notebooks/manual/basics/classification.ipynb (hunk 1 above)
Comment (NicolasDurrande) [reply doc/source/notebooks/manual/basics/classification.ipynb]: I'd like the code to be as neat as possible so I would be in favour of keeping it the way it is. If it happens to be too slow on the servers, I am happy to reduce the number of observation points, and to stop this optim before convergence.

File: doc/source/notebooks/manual/basics/classification.ipynb (hunk 2 above)
Comment (NicolasDurrande) [reply doc/source/notebooks/manual/basics/classification.ipynb]: done

**Pull request merged**